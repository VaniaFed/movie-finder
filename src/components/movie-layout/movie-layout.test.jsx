import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router';
import { MovieLayout } from './index';

describe('<MovieLayout />', () => {
    const movies = [];
    const tree = renderer
        .create(
            <MemoryRouter>
                <MovieLayout movies={movies} />
            </MemoryRouter>
        )
        .toJSON();
    it('should be rendered with movies', () => {
        expect(tree).toMatchSnapshot();
    });
});
