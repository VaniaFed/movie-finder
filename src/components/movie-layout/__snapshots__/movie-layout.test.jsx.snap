// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MovieLayout /> should be rendered with movies 1`] = `
<div
  className="movies"
/>
`;
